src/components/Property/Contacts/ContactsTable.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import ContactsRow from './ContactsRow'
+
+const ContactsTable = ({ contacts }) => {
+  return (
+    <table className="govuk-table">
```

Comment:
You may have missed it but Andrew has added some reusable <Table> <Thead> <TBody> components etc. Going forward might be worth using this (so we don't have to repeat all the generic classnames) but I know older table components don't have this updated yet so could address as one PR later

Reply:
Thanks. I'm going to leave this for now and I think we can address as a sweep later.